direct-query-core/src/main/java/org/opensearch/sql/prometheus/exception/PrometheusClientException.java

```diff
@@ -1,11 +1,9 @@
 /*
- *
- *  * Copyright OpenSearch Contributors
- *  * SPDX-License-Identifier: Apache-2.0
- *
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.opensearch.sql.prometheus.exceptions;
+package org.opensearch.sql.prometheus.exception;
 
 import org.opensearch.sql.datasources.exceptions.DataSourceClientException;
```

Comment:
this should be `package org.opensearch.sql.datasource.client.exceptions.DataSourceClientException;`? i think we should remove one of the two DataSourceClientException